src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -363,6 +363,15 @@ void
 CreateDistributedTable(Oid relationId, Var *distributionColumn, char distributionMethod,
 					   char *colocateWithTableName, bool viaDeprecatedAPI)
 {
+	List *fKeyCreationCommandsRelationInvolved = NIL;
+	if (IsCitusTableType(relationId, CITUS_LOCAL_TABLE))
```

Comment:
needs comment explaining the hack we are doing

Reply:
8879e44ae0ae844979317c213ffcb18bcfcb2548